tools/generate-raddstoreexpminusmax-test.py

```diff
@@ -27,65 +27,95 @@
 
 
 def split_ukernel_name(name):
-  match = re.fullmatch(r"xnn_(f16|f32)_raddstoreexpminusmax_ukernel__(.+)_u(\d+)(_acc(\d+))?", name)
+  match = re.fullmatch(r"xnn_(f16|f32)_raddstoreexpminusmax_ukernel__(.+)_u(\d+)(v)?(_acc(\d+))?", name)
   if match is None:
     raise ValueError("Unexpected microkernel name: " + name)
   elements_tile = int(match.group(3))
+  vector_tile = bool(match.group(4))
 
   arch, isa, assembly = xnncommon.parse_target_name(target_name=match.group(2))
-  return elements_tile, arch, isa
+  return elements_tile, vector_tile, arch, isa
 
 
 RADDSTOREEXPMINUSMAX_TEST_TEMPLATE = """\
-TEST(${TEST_NAME}, elements_eq_${ELEMENTS_TILE}) {
+TEST(${TEST_NAME}, elements_eq_${ELEMENTS_TILE}${ELEMENTS_SUFFIX}) {
   $if ISA_CHECK:
     ${ISA_CHECK};
   RAddStoreExpMinusMaxMicrokernelTester()
-    .elements(${ELEMENTS_TILE})
+    .elements(${ELEMENTS_TILE}${ELEMENTS_SCALE})
     .Test(${TEST_FUNCTION}, ${INIT_FUNCTION});
 }
 
 $if ELEMENTS_TILE > 1:
-  TEST(${TEST_NAME}, elements_div_${ELEMENTS_TILE}) {
+  TEST(${TEST_NAME}, elements_div_${ELEMENTS_TILE}${ELEMENTS_SUFFIX}) {
```

Comment:
This requires quite a lot of changes and poor coverage if tested on other platforms.  Its also not very important to RVV, since there is no remainder code to be tested.  I wonder if it can be simplified?

Reply:
> This requires quite a lot of changes and poor coverage if tested on other platforms. 

Agree.
We cannot express the idea of using `LMUL=m${ELEMENTS_TILE}` without `${ELEMENTS_SUFFIX}` on test name.